### PR TITLE
link `at` and `with` together

### PR DESCRIPTION
**Pull request by Rudxain**
### Description

Added `Array.prototype.at` link to `Array.prototype.with`'s "See also" section, and vice-versa.

### Motivation

Since both methods have the same relative-indexing behavior, and they are the "counterparts" for the absolute-indexed bracket-notation, they are closely related.

A user searching for one, may want to know about the other.

### CODE REVIEW

**Review by Josh-Cena — COMMENTED** [review]
Could you add a sentence to the description as well, about how they are basically the equivalents of using bracket notation for read-write?

**Comment by Rudxain** [comment]
> Could you add a sentence to the description as well, about how they are basically the equivalents of using bracket notation [...]?

WDYM? they already have it. Although `with` has it in the 1st paragraph, not the desc.

Do you mean adding it to the "See also" section?

**Comment by Josh-Cena** [comment]
Sorry, I mean the following sentence:

> By combining `at()`/`with()` with `with()`/`at()`, you can both read and write the array with negative indices.

**Comment by Rudxain** [comment]
@Josh-Cena Done!

File: files/en-us/web/javascript/reference/global_objects/array/with/index.md [hunk 1]
```diff
@@ -37,6 +37,8 @@ A new array with the element at `index` replaced with `value`.
 
 The `with()` method changes the value of a given index in the array, returning a new array with the element at the given index replaced with the given value. The original array is not modified. This allows you to chain array methods while doing manipulations.
 
+By combining this with {{jsxref("Array.prototype.at()", "at()")}}, you can both write and read (respectively) the array with negative indices.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/array/with/index.md:40]:
```suggestion
By combining `with()` with {{jsxref("Arra/at", "at()")}}, you can both write and read (respectively) an array using negative indices.
```

File: files/en-us/web/javascript/reference/global_objects/array/at/index.md [hunk 2]
```diff
@@ -32,6 +32,8 @@ The `at()` method is equivalent to the bracket notation when `index` is non-nega
 
 The usual practice is to access {{jsxref("Array/length", "length")}} and calculate the index from that — for example, `array[array.length - 1]`. The `at()` method allows relative indexing, so this can be shortened to `array.at(-1)`.
 
+By combining this with {{jsxref("Array.prototype.with()", "with()")}}, you can both read and write (respectively) the array with negative indices.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/array/at/index.md:35]:
```suggestion
By combining `at()` with {{jsxref("Array/with", "with()")}}, you can both read and write (respectively) an array using negative indices.
```

**Review by Josh-Cena — APPROVED** [review]
LGTM, thanks :)

**Comment by ChristianCCastro** [comment]
Muchas gracias amigos.  Creo que 6a me activaron una tarjeta.       Solo
compro comida   y me activo y le mandona mis hijos gracias

**Pull request merged**